module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -173,6 +173,7 @@ class KohaRest extends \VuFind\ILS\Driver\AbstractBase implements
         'gonenoaddress' => 'patron_status_address_missing',
         'debarred' => 'patron_status_card_blocked',
         'debt' => 'renew_debt',
+        'recalled' => 'Item requested - due date changed - return on due date',
```

Comment:
This should be a translation code like `ILSMessages::renewal_recalled`. We are now adding ILS message translations to ILSMessages text domain, so the actual text would be in e.g. `languages/ILSMessages/en.ini`.